clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/ops/servergroup/AbstractEnableDisableKubernetesAtomicOperation.groovy

```diff
@@ -159,13 +162,19 @@ abstract class AbstractEnableDisableKubernetesAtomicOperation implements AtomicO
     null // Return nothing from void
   }
 
-  Boolean supportsEnableDisable() {
-    switch(description.kind) {
+  boolean supportsEnableDisable(KubernetesV1Credentials credentials, String serverGroupName, String namespace) {
+    def controllerKind = description.kind
+    if (!description.kind) {
+      controllerKind = credentials.clientApiAdaptor.getControllerKind(serverGroupName, namespace, null)
```

Comment:
What if there is more than one controller with this name?

Reply:
I can add to default, but it is just print out the message.